From 6e1fd09f7bc131c8f16d9cc43e2455ba4650c651 Mon Sep 17 00:00:00 2001
From: Carlos Santos <casantos@datacom.com.br>
Date: Sat, 3 Nov 2018 08:25:58 -0300
Subject: [PATCH] Fix audispd path in auditd.conf

audispd is installed at /usr/sbin but the configuration file pointed
to /sbin, causing auditd to fail on startup.

This patch cannot be sent upstream because audispd does not exist
anymore on the master branch (it was merged to auditd).

Signed-off-by: Carlos Santos <casantos@datacom.com.br>
---
 init.d/auditd.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init.d/auditd.conf b/init.d/auditd.conf
index 4dcda83..998904f 100644
--- a/init.d/auditd.conf
+++ b/init.d/auditd.conf
@@ -13,7 +13,7 @@ max_log_file = 8
 num_logs = 5
 priority_boost = 4
 disp_qos = lossy
-dispatcher = /sbin/audispd
+dispatcher = /usr/sbin/audispd
 name_format = NONE
 ##name = mydomain
 max_log_file_action = ROTATE
-- 
2.17.1

